core/src/main/java/org/apache/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -190,7 +207,11 @@ public static boolean needApplyAllAuthorizationPlugin(SecurableObject securableO
     return false;
   }
 
-  private static boolean supportsSingleAuthorizationPlugin(MetadataObject.Type type) {
+  private static boolean needApplyAllAuthorizationPlugins(MetadataObject.Type type) {
```

Comment:
Change `needApplyAllAuthorizationPlugins()` to `needApplyAuthorizationAllCatalogs()` ?

Reply:
OK. Done.